### PR TITLE
Add create colony tests

### LINKED ISSUE #3221: Add tests to create a colony
Add tests to create a user & a colony

### PR DESCRIPTION
**Pull request by chinins**
## Description

Tis PR adds tests to create a colony.

To run tests you need to change `skipInitTests` in the cypress config file to `false`. This is because this test should be run only once (unless you work on this specific issue). 

**New stuff** ✨

- add new tests

**Changes** 🏗

- update cypress config

resolves #3221

### CODE REVIEW

**Review by arrenv — APPROVED** [review]
@chinins Aside from having to fiddle with timeouts and retrying a bunch of times to get it to run on my machine, it looks to be working!

One comment, would it be better to use the `getBySel` as opposed to `findByText` to avoid issues when button text may be changed?

**Review by ArmandoGraterol — COMMENTED** [review]
I also have the same doubt as Arren, wouldn't it be better if we use getBySel instead of findByText to select?

**Comment by chinins** [comment]
@arrenv, @ArmandoGraterol, I actually really like doing selectors based on text where possible. I think it's just easier to understand and also easier to write... If we were to have changes then we should be running tests anyway, and that would be a very easy update there.
I am of course open to change my code but I just really prefer the text where possible...

**Comment by chinins** [comment]
> I also have the same doubt as Arren, wouldn't it be better if we use getBySel instead of findByText to select?
> 
> **EDIT:**
> 
> Also I had to make a modification in the code to actually run the test without failing:
> 
> ```
>       cy.getBySel('colonyTokenSymbol', { timeout: 180000 }).should(
>         'have.text',
>         nativeToken,
>       );
> 
>       cy.url().should('eq', `${Cypress.config().baseUrl}/colony/${name}`);
> ```
> 
> For me, the `timeout` that the `cy.url().should()` has is not working and therefore is taking the default timeout which is not enough in my case

I wonder why timeout is not working for you in that case... I will switch the 2 tests around if that works better.

**Comment by arrenv** [comment]
> @arrenv, @ArmandoGraterol, I actually really like doing selectors based on text where possible. I think it's just easier to understand and also easier to write... If we were to have changes then we should be running tests anyway, and that would be a very easy update there. I am of course open to change my code but I just really prefer the text where possible...

Of course, it is completely your call, however here is some additional thinking as to why I feel I prefer the `getBySel`:
- if there is clashing text on the screen
- being able to find what is being selected in the code, for example, if I search the codebase for the text "continue" I get 134 matches. Where as you could narrow that down even with the same naming by searching for `data-test="continue"`
- if you are testing in a different language

**Comment by chinins** [comment]
> > @arrenv, @ArmandoGraterol, I actually really like doing selectors based on text where possible. I think it's just easier to understand and also easier to write... If we were to have changes then we should be running tests anyway, and that would be a very easy update there. I am of course open to change my code but I just really prefer the text where possible...
> 
> Of course, it is completely your call, however here is some additional thinking as to why I feel I prefer the `getBySel`:
> 
> * if there is clashing text on the screen
> * being able to find what is being selected in the code, for example, if I search the codebase for the text "continue" I get 134 matches. Where as you could narrow that down even with the same naming by searching for `data-test="continue"`
> * if you are testing in a different language

I didn't think of some of these points and I do admit that it makes sense. I will update accordingly.

**Review by dit7ya — COMMENTED** [review]
I tried running this but it failed at this stage. Should I manually increase the wait somewhere?

```console
Timed out retrying after 10000ms: Unable to find an element with the text: /create a new token/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
...
```

**Comment by chinins** [comment]
> I tried running this but it failed at this stage. Should I manually increase the wait somewhere?
> 
> ```
> Timed out retrying after 10000ms: Unable to find an element with the text: /create a new token/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
> ...
> ```

I can add a timeout to this. Can you test and tell me how long is needed.

**Comment by chinins** [comment]
> I tried running this but it failed at this stage. Should I manually increase the wait somewhere?
> 
> ```
> Timed out retrying after 10000ms: Unable to find an element with the text: /create a new token/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
> ...
> ```

I've add the timeout. You can change it if it's not enough.

**Comment by chinins** [comment]
@ArmandoGraterol @arrenv i've updated the selectors.

**Review by arrenv — COMMENTED** [review]
@chinins looks good to me :)

File: cypress.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
     "viewportWidth": 1250,
     "viewportHeight": 840,
     "colony": {
-        "name": "a",
-        "nativeToken": "A"
+        "name": "jaya",
```
[the remark this answers is not in the document]
Comment (ArmandoGraterol) [reply cypress.json:9]: That's okay, jaya also sounds like a cool name :)

**Review by ArmandoGraterol — APPROVED** [review]
Nice work! It's working as expected :ok_hand:

**Pull request merged**